lib/jxl/enc_ac_strategy.cc

```diff
@@ -1168,6 +1174,8 @@ void ProcessRectACSNew(PassesEncoderState* JXL_RESTRICT enc_state,
   // don't overlap.
   uint8_t priority[64] = {};
   for (auto tx : kTransformsForMerge) {
+    bool fool = false;
+    if (fool) continue;
```

Comment:
looks like temporary disabling this loop, want to keep this code this way?

Reply:
haha, yes, I'll clean that up -- disabling code with if (false) or #if 0 and many other things doesn't work any more with our compiler warnings